### DISCUSSION TITLE
useSwr in next js 13 appdir

### DISCUSSION THREAD

**Discussion by Bravemobin** [vercel/swr#2859] [Q&A] [answered]
I can use useswr just in client component but i want use this in server component
i know hooks cant run in server component but why SWR emphasis i can use this library in SSR

**Comment by ananuness** [comment, marked as answer]
Hii! I don't know if you already found the answer, but it's good to remember that you can add Client components to your Server components! Therefore, it is good to separate any block of code that needs the "use client" in another file and then import it into the Server component.
